conjure-java-core/src/test/java/com/palantir/conjure/java/types/SafetyEvaluatorTest.java

```diff
@@ -150,7 +191,7 @@ void testMapSafetyUnsafeValue() {
                 .types(SAFE_ALIAS)
                 .types(UNSAFE_ALIAS)
                 .build();
-        ConjureDefinitionValidator.validateAll(conjureDef);
+        ConjureDefinitionValidator.validateAll(conjureDef, SafetyDeclarationRequirements.ALLOWED);
```

Comment:
this method changed between the previous conjure version and the one I bumped to